test/e2e/case20_delete_objects_test.go

```diff
@@ -162,32 +161,31 @@ var _ = Describe("Test status fields being set for object deletion", func() {
 
 			deleteConfigPolicies(policies)
 
-			Consistently(func() interface{} {
+			utils.Kubectl("delete", "pod", case20PodName, "-n", "default", "--ignore-not-found")
+
+			Eventually(func() interface{} {
```

Comment:
@JustinKuli what are your thoughts here about the change from `Consistently` to `Eventually`?

Reply:
Actually, I didn't see that this was in the clean up so I think this should be fine.